test/ui/test_spec.js

```diff
@@ -96,29 +94,42 @@ describe('UI tests', function() {
     });
 
     it('appears correctly before any actions have been taken', function(done) {
-      return screenshotAndCompare('body.png', 'body', done);
+      screenshotAndCompare('body.png', 'body')
```

Comment:
This done wasn't adding anything so I took it out completely. Its sole purpose was making sure we wait for the screenshot to be compared and written to disk (if incorrect) before continuing, but we don't need that for the correctness of the tests, we only need to make sure the `driver.takeScreenshot` call is done.